### DISCUSSION TITLE
What features do you want to add to the next version of Mojo 🔥 0.8.0 🙋

### DISCUSSION THREAD

**Discussion by Eprahim-taha** [modular/modular#1698] [General]
Windows support. 

What about you? What features are you thinking of adding in your opinion❓

**Comment by josiahls** [comment]
Coroutine / Generator / Classes. I want to experiment with using mojo to build a simple data pipeline. I think the killer functionality out of mojo would be the type safety that I can build into my pipeline

**Reply by Brian-M-J** [reply]
>Coroutine

Mojo already has [coroutines](https://docs.modular.com/mojo/stdlib/builtin/coroutine.html), but the API is very low level.

**Reply by josiahls** [reply]
Ah interesting, yeah I guess Im looking for the yield / yield from API

**Comment by melodyogonna** [comment]
Not for 0.8... though wouldn't be bad if it makes it to 0.8. But the feature I'm most looking forward to is Swift-like extensions.

**Comment by ekbrown** [comment]
Regular expression support:
```
import re
re.findall()
re.finditer()
re.search()
re.sub()
```

**Comment by ghost** [comment]
i want #1751 resolved tbh

**Comment by Brian-M-J** [comment]
Now that the Mojo playground is being deprecated, I really hope native Windows support arrives in the next release.

**Comment by alfie-nsugh** [comment]
C++ interoperability would be very nice.

**Comment by manatlan** [comment]
I want https://github.com/modularml/mojo/issues/1216 resolved tbh ;-)

BTW, the new version is "mojo 24.1.0 (55ec12d6)" (not 0.8!), and it's 2x slower than the previous "0.7"

And while reading https://www.modular.com/blog/mojo-vs-rust-is-mojo-faster-than-rust, in my case, mojo is 10x slower than rust ;-(

see latests results : (updated every day)
https://github.com/manatlan/sudoku_resolver/blob/master/RESULTS.md
